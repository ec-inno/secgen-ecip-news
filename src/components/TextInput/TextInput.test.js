import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from './TextInput';

describe('TextInput', () => {
  it('renders correctly', () => {
    const props = { id: 'foo' };
    const tree = renderer.create(<TextInput {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
