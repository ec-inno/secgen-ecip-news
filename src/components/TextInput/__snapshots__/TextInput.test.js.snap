// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput renders correctly 1`] = `
<div
  className="ecl-form-group ecl-form-group--text-input"
>
  
  
  <input
    className="ecl-text-input"
    disabled={false}
    id="foo"
    required={false}
    type="text"
  />
</div>
`;
